README.md

```diff
@@ -59,6 +59,19 @@ Every voice is important and every idea is valuable. If you have something on yo
   - [mysql_user](https://docs.ansible.com/ansible/latest/collections/community/mysql/mysql_user_module.html)
   - [mysql_variables](https://docs.ansible.com/ansible/latest/collections/community/mysql/mysql_variables_module.html)
 
+
+## Releases Support Timeline
+
+In https://github.com/ansible-collections/community.mysql/discussions/537 it has been decided to maintain each major release (1.x.y, 2.x.y, ...) for two years after the release date.
```

Comment:
```suggestion
It has been [decided](https://github.com/ansible-collections/community.mysql/discussions/537) to maintain each major release (1.x.y, 2.x.y, ...) for two years after the release date.

The current major version is maintained at least until the next major version release, in case it takes longer than two years.
```